cli/src/semgrep/output.py

```diff
@@ -311,7 +311,15 @@ def output(
                 for err in self.semgrep_structured_errors
                 if SemgrepError.semgrep_error_type(err) == "SemgrepCoreError"
             ]
-            paths = {Path(err.core.location.path) for err in semgrep_core_errors}
+            paths = {
```

Comment:
What is this counting?

Reply:
Theoretically the number of lines that are skipped (but practically some number between 0 and "All", depending on which error comes last).

Reply:
Can you rename the variable to make that more clear? Can we also rename the instance variable this gets sent to

Reply:
And a comment explaining the edge cases  / why this is only theoretical and not actual number of lines that are skipped

Reply:
I got rid of all those edge cases in 06baab9